.repo-metadata.json

```diff
@@ -1,14 +1,16 @@
 {
   "name": "pubsublite",
   "name_pretty": "Cloud Pub/Sub Lite",
-  "product_documentation": "n/a",
-  "api_description": "n/a",
+  "product_documentation": "https://cloud.google.com/pubsub",
```

Comment:
This should be "https://cloud.google.com/pubsub/lite". It will go public next week.

Reply:
The full link to the docs will be be https://cloud.google.com/pubsub/lite/docs

https://cloud.google.com/pubsub links to the marketing page, and Pub/Sub and Pub/Sub Lite will share a marketing page, so https://cloud.google.com/pubsub/lite will redirect to https://cloud.google.com/pubsub